reports.py

```diff
@@ -6,6 +6,23 @@
 import re
 
 import database
+import html_writer
+
+
+CAT_2_TEAM = {
```

Comment:
what is CAT_2_TEAM?

Reply:
category to team. It helps prompt the triage person for a good team to assign an ancient issue to.

Reply:
I would rather remove it soon and replace by flask. Then there will be no real code. Maybe we'll have a new module as a sibling of reports and garden.